modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/util/NameUtils.java

```diff
@@ -132,28 +132,6 @@ private static void uncountable(String word)
     }
 
     /**
-     * Transforms a hyphenized value into a camel case one.
-     * <p>
-     * For example:
-     * {@code message-processor} would be transformed to {@code messageProcessor}
-     *
-     * @param hyphenizedValue a String in hypenized form
-     * @return the {@code hyphenizedValue} in camel case format
-     */
-    public static String camel(String hyphenizedValue)
```

Comment:
I'll actually move the entire thing to the DSL package because there's another PR in the way which will require this entirely without depending on extesions-spring-support